packages/affiliate/src/Affiliate.php

```diff
@@ -1,25 +1,33 @@
 <?php
+namespace Automattic\Jetpack\Partners;
 
-if ( ! defined( 'ABSPATH' ) ) {
-	// Exit if accessed directly
-	exit;
+use Jetpack;
+
+if ( ! defined( 'ABSPATH' ) || ! is_admin() ) {
```

Comment:
These statements shouldn't be necessary, as ideally such files are expected to only contain declarations of classes.

Reply:
Right.